.claude/scripts/io-performance-analyzer/merge_queue.py

```
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""Merge disk await time from pmrep CSV into the main IO CSV.
+
+Reads the pmrep CSV (with disk.dev.await per-device columns), aligns
+timestamps to the existing IO CSV rows, and adds an 'await' column
+(max across devices). Writes in-place.
+"""
+
+import csv
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)